docs/console/company-configuration/providers/extensions/orchestrator-generator.md

```diff
@@ -0,0 +1,270 @@
+---
+id: orchestrator-generator
+title: Orchestrator Generator
+sidebar_label: Orchestrator Generator
+---
+
+![Orchestrator Generator](img/orchestrator-generator.png)
+
+:::info
+
+An Orchestrator Generator can only be used when the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled.
+
+:::
+
+One of the main features of Mia-Platform is the automatic generation of the **orchestrator's files**, ready to **be deployed**; these files are saved in the **configuration repository**. For instance, the Console could generate Kubernetes files.
```

Comment:
```suggestion
One of the main features of Mia-Platform is the automatic generation of the **orchestrator's files**, ready to **be deployed**; these files are saved in the **configuration repository**. For instance, the Console can generate Kubernetes files and deploy them on your runtime based on the Design configuration you provided.
```


maybe link the Design word to the proper page?

Reply:
done, with the link